docs/courses/editing-courses.md

```diff
@@ -1,183 +1,153 @@
-# Creating and editing LibreLingo courses
+# Editing LibreLingo courses
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
+Thank you for your interest in contributing to LibreLingo. This document is here to guide you for editing or translating existing courses.
 
-A simplified example of the directory structure of a LibreLingo course:
+To get a better understanding, we recommend you read the [course basics](README.md#basics) first.
 
+---
+
+**Confused?**
+[Ask people](https://github.com/kantord/LibreLingo/discussions) on GitHub Discussions.
+
+---
+
+**Table of Contents:**
+- [Setup](#setup)
+  - [Pulling code and branching](#pulling-and-branching)
+  - [Pushing code and creating Pull Request](#pushing-and-pr)
+  - [Following up with responses](#following-up-pr)
+- [Translating based on an existing course](#translating)
+- [Editing existing courses](#editing-existing)
+
+## Setup
+
+<a id="pulling-and-branching"></a>
+### Pulling code and branching
```

Comment:
I would probably move this to a separate file and rename it to something like "Editing courses in a text editor no your computer".

BUt it doesn't have to be done for this PR

Reply:
This seems like a good idea. Since some might prefer to work on their own machines. If this is made separate, I think we can also add instructions for a Git GUI, which I didn't include being afraid that it would be too much details and make it complex to navigate. Maybe on a separate PR would be great.

I thought including instructions like this was a good idea since we would need to generate json files based on the course yaml anyway. I am not sure if it can be done through CI or something. If this can be done, I think using a CI/CD for this kind of task instead would make this a lot easier both for devs and contributors.

Reply:
Even if CI/CD is implemented, I think we should add the instructions for yaml generation on that separate docs, because course writers may need to test changes locally on the webapp.

Reply:
> I am not sure if it can be done through CI or something. If this can be done, I think using a CI/CD for this kind of task instead would make this a lot easier both for devs and contributors.

Definitely, that's a long term goal